old_data/sensor_reading_to_s3/transform.py

```diff
@@ -0,0 +1,55 @@
+"""Transform the data ready to upload to s3"""
+import os
+from os import environ as ENV
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+from dotenv import load_dotenv
+
+
+def read_csv():
+    df = pd.read_csv("data/sensor_reading.csv")
+    return df
+
+
+def create_directories(base_dir="c17-james-plant-bucket"):
```

Comment:
Just a missing typehint.
```suggestion
def create_directories(base_dir: str = "c17-james-plant-bucket"):
```